charts/gsp-cluster/values.yaml

```diff
@@ -730,8 +730,8 @@ concourse:
     emptyDirSize: 60Gi
     resources:
       requests:
-        cpu: 150m
-        memory: 2Gi
+        cpu: 2000m
+        memory: 12Gi
```

Comment:
they currently live on nodes with 8Gi memory, is this necessary? if we want to ensure it gets a node all to itself, this will still leave 4Gi worth of memory unused which could potentially be used to run other pods

Reply:
between the kublet's "reserving" of some node memory for system use and the daemonset's memory usage ... 12Gi was the practical max amount that didn't result in pod becoming un-schedulable when doing a similar "try and take most of a node" for prometheus pods.

whether this is acceptable / "good enough" to count as concourse workers getting their own node is a question worth discussing.

context: I don't believe there is an ADR that says workers must have their own VM... but original designs had concourse in a separate cluster, so separate VMs was previously implicit. when we moved to single cluster, we maintained the VM isolation to avoid having to discuss isolation again.